src/test_util/components.rs

```diff
@@ -53,6 +53,7 @@ pub const FILE_SOURCE_TAGS: [&str; 1] = ["file"];
 /// The most basic set of tags for sinks, regardless of whether or not they push data or have it pulled out.
 pub const SINK_TAGS: [&str; 1] = ["protocol"];
 
+///
```

Comment:
Would love input on what this sink tag is for

Reply:
It is possible to configure Vector to emit the sink metrics tagged by `source` and `service`:

```yml
tags:
  emit_service: true
  emit_source: true
```

- `source` is the name of the source component that created the event.
- `service` is the service, which is whatever field has the meaning of `service`. This can depend on where the event came from. Syslog uses (I think) the AppName field to represent the service, other sources define this as different fields.

With this on it can help to track the flow of data through Vector more accurately.